utkdigitalinitiatives/utk_isl_xml_forms
# Address CON-34.

**JIRA Ticket**: [CON-34](https://jira.lib.utk.edu/browse/CON-34)

# What does this Pull Request do?

This adds an Islandora XML form specifically for new issues of Football Programs.

# What's new?

A new form has been added.

# How should this be tested?

* Start up your Vagrant
* Import the new form
* Associate it with your favorite content model
* Create a new object of your favorite content model and select the newly associated form
* Now edit the record and make sure it reads the values in correctly
* Now open the MODS and look at the record.  Does all look good? Are there blank nodes?

# Additional Notes:
This assumes that new records should look like the old ones.  The data model hasn't changed at all. 

Also, one of the dates will need to be handled in the post-process transform.  I'll create a separate ticket and pull request for that.

# Interested parties
@mlhale7
